zerver/models.py

```diff
@@ -3516,6 +3516,7 @@ class AbstractAttachment(models.Model):
     # If the path of a file is http://localhost:9991/user_uploads/a/b/abc/temp_file.py
     # then its path_id will be a/b/abc/temp_file.py.
     path_id = models.TextField(db_index=True, unique=True)
+    tus_file_id = models.CharField(max_length=60, db_index=True, unique=True, null=True)
```

Comment:
Please add a block comment here documenting concisely but with key details included the semantics of this field.

(How was the `max_length` picked, BTW?)

Reply:
I think the answer here is that this is a file ID that is only used for allowing Zulip to look up the `path_id` for an attachment that was uploaded using `TUS` to return to the client. I don't love this being here, but I suppose it's not hard to delete a nullable field with a default of null from a table once it's not required, so I can live with this if we can't wait for tusv2.